chart/templates/_initContainerCachedAssets.tpl

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 The HuggingFace Authors.
+
+{{- define "initContainerCachedAssets" -}}
+- name: prepare-assets
+  image: ubuntu:focal
+  imagePullPolicy: {{ .Values.images.pullPolicy }}
+  command: ["/bin/sh", "-c"]
+  args:
+  - chown {{ .Values.uid }}:{{ .Values.gid }} /mounted-path;
+  volumeMounts:
+  - mountPath: /mounted-path
```

Comment:
Why is `/mounted-path` hardcoded here ? It may cause issues to have the same as the one used with `assets.subpath` ?

Reply:
no, I don't think so, but we have to change the name above